<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginview</class>
 <widget class="QWidget" name="loginview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loginview</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_roomnumber">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>134</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic/msg_2.jpg);
color:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>194</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic/msg_2.jpg);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_login">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>135</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic/dl.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic/denglu.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_msg">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>237</y>
     <width>364</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic/msg_1.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>lineEdit_roomnumber</zorder>
  <zorder>lineEdit_id</zorder>
  <zorder>Button_login</zorder>
  <zorder>lineEdit_msg</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
